File: 06_sorting/triangle.py
def solution(A):
    l = len(A)

    if l < 3: # range of n starts from 0
        return 0

    A.sort()

    for i in xrange(l-2):
        if A[i] + A[i+1] > A[i+2]:
            return 1
    return 0

File: 06_sorting/triangle.cpp
#include "vector"
#include "algorithm"

using namespace std;

int solution(vector<int> &a) {
  int n = a.size();
  sort(a.begin(), a.end());
  for (int i = 0; i < n - 2; i++) {
    if ((long long)a[i] + a[i + 1] > a[i + 2]) {
      return 1;
    }
  }
  return 0;
}
